.circleci/config.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     executor: << parameters.e >>
     steps:
       - checkout
+      - run: composer validate
```

Comment:
I was able to get both `build_and_test-1` and `build_and_test-2` to succeed in local CI builds by removing the `compose validate` command. Also not sure if that is a good solution, and would like your thoughts.

Reply:
@jkowalleck Thanks so much for your explanations - it helps me get a clue!
I think I have successfully resolved the merge conflict and preserved your intention to validate things in the build - but please take a look to make sure I'm on track.

FWIW, all the local CI builds appear happy.

Reply:
@bhamail 

> I think I have successfully resolved the merge conflict [...]

what merge conflicts? there were no merge conflicts.
I rebased this branch onto "main" without any issues and pushed it to github.  see https://github.com/sonatype-nexus-community/bach/pull/27#event-4615170419

i set this PR to be a draft, until things get sorted out.